index.html

```diff
@@ -319,9 +331,129 @@ <h3>Terminology</h3>
       <h2>Data Model</h2>
 
       <p>
-The following sections outline the data model for this document.
+There are numerous ways to express status information associated with digital
+credentials. Some of these mechanisms include Certificate Revocation Lists (CRL)
+[[?RFC5280]], the Online Certificate Status Protocol (OCSP) [[?RFC2560]], Bloom
+Filters [[?RFC8932]], and cryptographic accumulators [[?ALLOSAUR]]. This
+specification optimizes for a variety of requirements that are different from
+other mechanisms. These requirements include:
       </p>
 
+      <table class="simple" id="#pfatable">
+        <thead>
+        <tr>
+            <th style="text-align: center;">Technology</th>
+            <th style="text-align: center;">CRL</th>
+            <th style="text-align: center;">OCSP</th>
+            <th style="text-align: center;">Bloom</th>
+            <th style="text-align: center;">Accumulator</th>
+            <th style="text-align: center;">Bitstring</th>
+        </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+Provides tunable group privacy
+            </td>
+            <td class="supported">✓</td>
+            <td class="missing">✗</td>
+            <td class="supported">✓</td>
+            <td class="supported">✓</td>
+            <td class="supported">✓</td>
+          </tr>
+          <tr>
+            <td>
+Does not require signed assertion for each credential
+            </td>
+            <td class="supported">✓</td>
+            <td class="missing">✗</td>
+            <td class="supported">✓</td>
+            <td class="supported">✓</td>
+            <td class="supported">✓</td>
+          </tr>
+          <tr>
+            <td>
+Resistant to [=issuer=] tracking when fetched by [=verifier=]
+            </td>
+            <td class="supported">✓</td>
+            <td class="missing">✗</td>
```

Comment:
```suggestion
            <td class="missing">?</td>
```
Can be supported when there is an OCSP server separate from the issuer

Reply:
Let's presume an ideal deployment of OCSP, as you say, and change this to "supported". I've changed your suggestion from a "?" to a "✓", and have committed it.